Dockerfile.spdk

```diff
@@ -0,0 +1,107 @@
+# syntax = docker/dockerfile:1.4
+ARG OS_RELEASE=9
+FROM registry.access.redhat.com/ubi$OS_RELEASE/ubi:latest as prebase
+ARG OS_RELEASE
+
+# Speed up dnf
+COPY <<EOF /etc/dnf/dnf.conf
+[main]
+gpgcheck=1
+installonly_limit=3
+clean_requirements_on_remove=True
+skip_if_unavailable=False
+best=False
+fastestmirror=True
+max_parallel_downloads=10
+tsflags=nodocs
+EOF
+
+#------------------------------------------------------------------------------
+FROM prebase as base_release_8
```

Comment:
You could reduce the complexity and drop CentOS 8, and only support CentOS 9

Reply:
Makes sense. That said, I hoped to eventually convince the SPDK folks to add this Dockerfile to their project root, so we basically run a `docker build github.com/spdk/spdk` and we don't have to worry about maintaining that (they already have a Dockerfile, but it's for testing purposes).